src/Klarna.Checkout/KlarnaCheckoutService.cs

```diff
@@ -83,16 +83,13 @@ public virtual Client GetClient(IMarket market)
             if (PaymentMethodDto != null)
             {
                 var connectionConfiguration = GetCheckoutConfiguration(market);
-                var connector = ConnectorFactory.Create(connectionConfiguration.Username, connectionConfiguration.Password, new Uri(connectionConfiguration.ApiUrl));
-                connector.UserAgent.AddField("Platform", "Episerver.Commerce_", typeof(EPiServer.Commerce.ApplicationContext).Assembly.GetName().Version.ToString(), new string[0]);
```

Comment:
We want to keep and send custom useragent headers. Is there anyway we can do that with the new package?